packages/office-ui-fabric-react/src/components/Tooltip/Tooltip.base.tsx

```diff
@@ -43,6 +43,7 @@ export class TooltipBase extends React.Component<ITooltipProps, any> {
       theme: theme!,
       className: className || (calloutProps && calloutProps.className),
       delay: delay!,
+      gapSpace: (calloutProps && calloutProps.gapSpace && -15 - calloutProps.gapSpace) || -15,
```

Comment:
we've already got defaultProps set, so you shouldn't need to check this, right? Does TS complain? Also, how about doing the -15 calc in the styles file.

Reply:
TS was complaining, I defaulted to using `gapSpace?` and setting a the default to 0 in the `styles`. I just pushed that change which I think makes the code a little bit cleaner.

Reply:
ah yeah, i was wondering if typescript would complain. Could always change it to have a default value in the destructuring, but no worries.